simulation/simple_sensor_simulator/package.xml

```diff
@@ -40,6 +40,7 @@
   <depend>boost</depend>
   <depend>eigen</depend>
   <depend>embree_vendor</depend>
+  <depend>glog</depend>
```

Comment:
```suggestion
  <depend>libgoogle-glog-dev</depend>
```

Please check rosdep.yaml when you add apt package dependency to ROS 2 package.

https://github.com/ros/rosdistro/blob/8384c11cfbd2455db9115cda80545ec6593887fa/rosdep/base.yaml#L3909